File-Formats/DocIO/word-to-image.md

```diff
@@ -137,6 +138,203 @@ using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResource
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-Image-conversion/Convert-Word-to-image).
 
+## Convert first page of Word to image
+
+You can convert a specific page of the Word document into an image and use it for a thumbnail.
+
+The following code example illustrates how to convert a first page in a Word document into an image.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using(WordDocument wordDocument = new WordDocument("Template.docx", FormatType.Docx))
+{
+    //Initialize the ChartToImageConverter for converting charts during Word to image conversion.
+    wordDocument.ChartToImageConverter = new ChartToImageConverter();
+    //Set the scaling mode for charts (Normal mode reduces the file size).
+    wordDocument.ChartToImageConverter.ScalingMode = ScalingMode.Normal;
+    //Convert the first page of the Word document into an image.
+    Image image = wordDocument.RenderAsImages(0, ImageType.Bitmap);
+    //Save the image as jpeg.
+    image.Save("WordToImage.jpeg", ImageFormat.Jpeg);
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Load an existing Word document.
+Using wordDocument As WordDocument = New WordDocument("Template.docx", FormatType.Docx)
+    'Initialize the ChartToImageConverter for converting charts during Word to image conversion.
+    wordDocument.ChartToImageConverter = New ChartToImageConverter()
+    'Set the scaling mode for charts (Normal mode reduces the file size).
+    wordDocument.ChartToImageConverter.ScalingMode = ScalingMode.Normal
+    'Convert the first page of the Word document into an image.
+    Dim image As Image = wordDocument.RenderAsImages(0, ImageType.Bitmap)
+    'Save the image as jpeg.
+    image.Save("WordToImage.jpeg", ImageFormat.Jpeg)
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//DocIO supports Word to image conversion in Windows Forms, WPF, ASP.NET and ASP.NET MVC platform alone
```

Comment:
//DocIO supports Word to image conversion in UWP application using DocIORenderer.

Check in UWP and add code snippet using DocIORenderer

Reply:
Ensured the code snippet using DocIORenderer in UWP application and added